src/app/_components/NavBar/LabelWrapper.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box } from '../../../ui/Box';
+import { Flex } from '../../../ui/Flex';
+import { Link } from '../../../ui/Link';
+import { NavItem } from './types';
+import { useIsDesktop } from './utils';
+
+export const LabelWrapper = ({ label, href, onClick }: NavItem) => {
+  const isDesktop = useIsDesktop();
+  return (
+    <Flex
+      alignItems={'center'}
+      borderRadius="xl"
+      width="full"
+      {...(isDesktop ? { _hover: { bg: 'hoverBackground' } } : {})}
```

Comment:
You can use [Chakra UI responsive style array syntax ](https://v2.chakra-ui.com/docs/styled-system/responsive-styles#the-array-syntax)to achieve this without custom hook or conditional style

Reply:
done